src/recipe.data.ts

```diff
@@ -1,5 +1,8 @@
 import mongoose, { Schema } from "mongoose";
 import { Recipe } from "./recipe";
+import { emptyRecipe } from "./emptyRecipe";
+
+type RecipePreview = Omit<Recipe, "id">;
```

Comment:
You declare this twice

Reply:
The file needs to be able to reference it.

Are you saying I should export it form the controller and call it that way?